### PR TITLE
updating the book name

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/14360

### CODE REVIEW

File: access_control/rbac.adoc [hunk 1]
```diff
@@ -521,4 +521,4 @@ To manage components of observability, view the following API RBAC table:
 | get, list, watch
 |===
 
-To continue to learn more about securing your cluster, see link:../risk_compliance/security_intro.adoc#security[Security].
+To continue to learn more about securing your cluster, see link:../governance/security_intro.adoc#security[Risk and compliance].
```
Comment (cadawson) [inline access_control/rbac.adoc:524]: Just making sure that you intended to leave the # as security, even though you changed the name to Risk and compliance. Won't break anything, but just making sure that was the intent.

File: governance/gatekeeper_policy.adoc [hunk 2]
```diff
@@ -114,4 +114,4 @@ spec:
 
 See https://github.com/open-cluster-management/policy-collection/blob/master/community/CM-Configuration-Management/policy-gatekeeper-sample.yaml[`policy-gatekeeper-sample.yaml`] for more details.
 
-Learn how to use {product-title-short} gatekeeper operator policy to install gatekeeper and create a {product-title-short} gatekeeper operator policy, see xref:../risk_compliance/create_gatekeeper.adoc#creating-a-gatekeeper-policy-from-the-console[Creating a gatekeeper policy from the console] for more details. Refer to xref:../risk_compliance/grc_intro.adoc#governance[Governance] for more topics on the security framework.
```
Comment (cadawson) [inline governance/gatekeeper_policy.adoc:117]: This looks like it is in italics in the preview. If it is not really, then I'm good.

**Review by cadawson — APPROVED** [review]
A couple of notes, but it looks like you caught them. I don't see anything scary in there. Thanks!

File: access_control/rbac.adoc (hunk 1 above)
Comment (dockerymick) [reply access_control/rbac.adoc:524]: @cadawson thanks for asking, yea I intended to keep the anchor as "security".

File: governance/gatekeeper_policy.adoc (hunk 2 above)
Comment (dockerymick) [reply governance/gatekeeper_policy.adoc:117]: Thanks, yea I went to the source and it didn't appear to be italic

**Pull request merged**